pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/NamespaceStatsAggregator.java

```diff
@@ -302,22 +315,22 @@ private static void getTopicStats(Topic topic, TopicStats stats, boolean include
                 });
     }
 
-    private static void printDefaultBrokerStats(PrometheusMetricStreams stream, String cluster) {
-        // Print metrics with 0 values. This is necessary to have the available brokers being
+    private static void printBrokerStats(PrometheusMetricStreams stream, String cluster, AggregatedBrokerStats brokerStats) {
```

Comment:
I think I misunderstood the original motivation for this PR. Is the issue that you're unable to sum `pulsar_topics_count` while ignoring the `namespace` label?

Reply:
When `exposeTopicLevelMetricsInPrometheus=true`, pulsar will expose topic partition level metrics. For getting the total number of topics, it is really possible to summarize the number of topics in all namespaces, and the number of namespaces is usually small.

But if you want to get the broker dimension other metrics,  example :  `pulsar_rate_in、pulsar_rate_out、pulsar_throughput_in、pulsar_throughput_out ` and so on,  will have to summarize the data of all topic partitions.

When there are few topic partitions, such as less than 10000,  it is very quick to directly summarize the metrics of topic dimensions. However, when the topic partition exceeds 100000, or even millions, it is likely that the result data cannot be queried from prometheus.

The structure of `pulsar_topics_count` metrics  in Prometheus is as follows:

`pulsar_topics_count{app="pulsar", cluster="pulsar-cluster", component="broker", namespace="mytenant/mynamespace", instance="xxx.xxx.xx.xx:8080", pod="pulsar-cluster-broker-1"}`

You can get the total number of topic through the following prometheus query:
`sum(pulsar_topics_count{cluster="$cluster",component="broker"})`


The structure of `pulsar_rate_in` metrics  in Prometheus is as follows:

`pulsar_rate_in{app="pulsar", cluster="pulsar-cluster", component="broker", namespace="10307612/default",instance="xxx.xxx.xx.xx:8080", pod="pulsar-cluster-broker-1", topic="persistent://mytenant/mynamespace/mytopic-partition-0"}`

Reply:
Sorry, I misunderstood your description before

The metrics name is still use the original name. To get the broker dimension metrics, we can set the `namespace` label as an empty string.

for example:
`pulsar_rate_in{namespace=""}`

After testing, queries are much faster than summarizing all topics. Even after the query time interval reaches several days, the results cannot be found by summarizing all topics, but the results can be found quickly by summarizing the broker metrics.

```
writeMetric(stream, "pulsar_topics_count", brokerStats.topicsCount, cluster);
writeMetric(stream, "pulsar_subscriptions_count", brokerStats.subscriptionsCount, cluster);
writeMetric(stream, "pulsar_producers_count", brokerStats.producersCount, cluster);
writeMetric(stream, "pulsar_consumers_count", brokerStats.consumersCount, cluster);
writeMetric(stream, "pulsar_rate_in", brokerStats.rateIn, cluster);
writeMetric(stream, "pulsar_rate_out", brokerStats.rateOut, cluster);
writeMetric(stream, "pulsar_throughput_in", brokerStats.throughputIn, cluster);
writeMetric(stream, "pulsar_throughput_out", brokerStats.throughputOut, cluster);
writeMetric(stream, "pulsar_storage_size", brokerStats.storageSize, cluster);
writeMetric(stream, "pulsar_storage_logical_size", brokerStats.storageLogicalSize, cluster);
writeMetric(stream, "pulsar_storage_write_rate", brokerStats.storageWriteRate, cluster);
writeMetric(stream, "pulsar_storage_read_rate", brokerStats.storageReadRate, cluster);
writeMetric(stream, "pulsar_msg_backlog", brokerStats.msgBacklog, cluster);
```

> I think I misunderstood the original motivation for this PR. Is the issue that you're unable to sum `pulsar_topics_count` while ignoring the `namespace` label?

Reply:
Sorry, I misunderstood your description before

The metrics name is still use the original name. To get the broker dimension metrics, we can set the `namespace` label as an empty string.

for example:
`pulsar_rate_in{namespace=""}`

After testing, queries are much faster than summarizing all topics. Even after the query time interval reaches several days, the results cannot be found by summarizing all topics, but the results can be found quickly by summarizing the broker metrics.

```
writeMetric(stream, "pulsar_topics_count", brokerStats.topicsCount, cluster);
writeMetric(stream, "pulsar_subscriptions_count", brokerStats.subscriptionsCount, cluster);
writeMetric(stream, "pulsar_producers_count", brokerStats.producersCount, cluster);
writeMetric(stream, "pulsar_consumers_count", brokerStats.consumersCount, cluster);
writeMetric(stream, "pulsar_rate_in", brokerStats.rateIn, cluster);
writeMetric(stream, "pulsar_rate_out", brokerStats.rateOut, cluster);
writeMetric(stream, "pulsar_throughput_in", brokerStats.throughputIn, cluster);
writeMetric(stream, "pulsar_throughput_out", brokerStats.throughputOut, cluster);
writeMetric(stream, "pulsar_storage_size", brokerStats.storageSize, cluster);
writeMetric(stream, "pulsar_storage_logical_size", brokerStats.storageLogicalSize, cluster);
writeMetric(stream, "pulsar_storage_write_rate", brokerStats.storageWriteRate, cluster);
writeMetric(stream, "pulsar_storage_read_rate", brokerStats.storageReadRate, cluster);
writeMetric(stream, "pulsar_msg_backlog", brokerStats.msgBacklog, cluster);
```

> I think I misunderstood the original motivation for this PR. Is the issue that you're unable to sum `pulsar_topics_count` while ignoring the `namespace` label?